configure.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+echo "Running configuration script"
+export DEBIAN_FRONTEND=noninteractive
+
+echo "US/Central" | sudo tee /etc/timezone
+dpkg-reconfigure --frontend noninteractive tzdata
+
+echo "Install essential software pacakges"
+apt-get -qq update
+apt-get -qq install -y wget htop wget build-essential zip software-properties-common gnupg awscli
+apt-get -qq install -y ansible
+echo $PERM_ENV  > /data/www/host.txt
+
+echo "Add custom sources"
+# Add mysql key
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5072E1F5
+# Add psql key
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+# Add custom sources
+cp $TEMPLATES_PATH/etc/apt/sources.list.d/* /etc/apt/sources.list.d/
+
+apt-get -qq update
+echo "Install web server tools"
```

Comment:
Similar comment re possible relevance of deps list to `torque` and `torque-sites`.

Reply:
It does look like php-mysql is needed by at least one competition, so I'm removing that dep from here and adding it to the mediawiki ansible task.